modules/filtering-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="filtering-unnecessary-advisor-recommendations_{context}"]
+= Filtering Insights advisor recommendations
+
+As an {product-title} cluster manager, you can filter the recommendations that are displayed on the recommendations list. By applying filters, you can reduce the number of reported recommendations and concentrate on your highest priority recommendations.
+
+The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the filter types and respective values. 
+
+.Prerequisites
+You are logged in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console].
+
+.Procedure
+. Navigate to *Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*.
+. In the left filter drop-down list, select the *Category* filter type.
+. Expand the right drop-down list and select the checkbox next to each category of recommendation you want to view. Leave the checkboxes for unnecessary categories clear.
```

Comment:
I'm still wondering if there's some way to avoid referring to "left" and "right". [The IBM guidelines say not to refer to locations using sensory elements](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements). Do these lists have a label or a heading of some kind that you can refer to?

Reply:
I've changed the first instance from "left filter drop-down list" to "main, filter-type drop-down list," and the second instance from "right drop-down list" to "filter-value drop-down list." It still sounds inelegant to me, but I'll take this to my team and ask how to make this better with respect to the ISG and your feedback.